src/core/models.py

```diff
@@ -1055,36 +1055,36 @@ def onetaskers(self):
                 users.append(user)
         return users
 
-    def all_editors(self):
-        press_editors = self.press_editors.all()
-        book_editors = self.book_editors.all()
+    def get_series_editor(self):
+        """Gets the editor of this book's series, if it is part of one.
 
+        Returns:
+            User: the book's series editor.
+        """
         if self.series:
-            series_editor = self.series.editor
-
-            if series_editor:
-                series_editor_list = [series_editor]
-                press_editor_list = [
-                    editor for editor in press_editors
-                    if not editor == series_editor_list[0]
-                ]
-            else:
-                series_editor_list = []
-                press_editor_list = [editor for editor in press_editors]
+            return self.series.editor
+        return None
```

Comment:
There is the school of thought which says, don't ever return None/NULL if you would otherwise expect a object of some description. If you can't find what you're looking for, return an object, like a dummy object, that way the caller doesn't need to take into account if it's NULL or not. This reduces the number of if statements you use, which is a good thing. In this case, it might be more trouble than it's worth, but I thought I'd mention it, just incase you've never heard of this idea.